projects/RabbitMQ.Client/client/impl/RabbitMQActivitySource.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using RabbitMQ.Client.Events;
+using RabbitMQ.Client.Framing.Impl;
+
+namespace RabbitMQ.Client
+{
+    internal class RabbitMQActivitySource
+    {
+        internal const string MessageId = "messaging.message.id";
+        internal const string MessageConversationId = "messaging.message.conversation_id";
+        internal const string MessagingOperation = "messaging.operation";
+        internal const string MessagingSystem = "messaging.system";
+        internal const string MessagingDestination = "messaging.destination.name";
+        internal const string MessagingDestinationKind = "messaging.destination.kind";
```

Comment:
New changes in specification, please remove this tag, based on https://github.com/open-telemetry/opentelemetry-specification/pull/3214

Reply:
Removed